filesmanager/filesmanager.py

```diff
@@ -547,6 +563,12 @@ def upload_file_to_directory(self, file, target_directory):
         )
         self.content_paths.append(file_path)
 
+    def find_temporary_file(self, file_name):
+        for file in self.temporary_uploaded_files:
+            if file.file.name == file_name:
+                return deepcopy(file)
```

Comment:
why a deepcopy?

Reply:
A shallow copy only copy high level attributes, while nested attributes like the file data itself will point to the same file object